ui/components/link.js

```diff
@@ -3,24 +3,35 @@ import React from 'react';
 
 import { Link as RouterLink, routes } from '../routes';
 
-export default function Link({ children, href, params, route, ...otherProps }) {
-  if (route) {
-    return (
-      <RouterLink params={params} route={route}>
-        <a {...otherProps}>{children}</a>
-      </RouterLink>
-    );
-  } else if (href) {
-    const props = {
-      ...otherProps,
-      href,
-    };
-    if (href.search(/^https|http|^\/\//) === 0) {
-      props.target = '_blank';
-      props.rel = 'noopener noreferrer';
-      props['aria-label'] = 'Link opens in a new window';
+export default class Link extends React.Component {
+  focus() {
+    if (this.el) {
+      this.el.focus();
     }
-    return <a {...props}>{children}</a>;
+  }
+
+  render() {
+    const { children, href, params, route, ...otherProps } = this.props;
+
+    if (route) {
+      return (
+        <RouterLink params={params} route={route}>
+          <a {...otherProps}>{children}</a>
+        </RouterLink>
+      );
+    } else if (href) {
+      const props = {
+        ...otherProps,
+        href,
+      };
+      if (href.search(/^https|http|^\/\//) === 0) {
+        props.target = '_blank';
+        props.rel = 'noopener noreferrer';
+        props['aria-label'] = 'Link opens in a new window';
```

Comment:
This is pre-existing code but it's bad code--see #655. I could quickly fix it in this PR by just removing the line if you want, let me know!

Reply:
Sounds good to me; I commented a bit more in that ticket.